docroot/profiles/custom/sitenow/sitenow.profile

```diff
@@ -378,6 +378,16 @@ function sitenow_config_split_prepare_form(EntityInterface $entity, $operation,
  * Custom node content type form defaults.
  */
 function _sitenow_node_form_defaults(&$form, $form_state) {
+  // Collapse inner SEO field groups.
```

Comment:
Testing now. One nit though: should this be in sitenow_seo?

Reply:
Yeah probably but it is unclear where the line is. I can move it when I get a chance.

Reply:
Yea I agree. Like I said, a nitpick. Seems like more and more is moving aware from the profile into modules which is probably better long (long) term. :)

Reply:
This is somewhat related to #2878, which explains the rationale of why we should move this to `uiowa_core` eventually. Basically, all the modules that are enabled are dependencies of the profile, but then you have modules that might make use of core functionality, and so the profile is also technically a dependency of the module. We should be better about actually tracking those dependencies where they exist. I very much agree that this is future work though.

Reply:
Moved: cd33adb